pkg/webui/console/views/gateway-overview/gateway-overview.js

```diff
@@ -58,6 +92,11 @@ export default class GatewayOverview extends React.Component {
     collaboratorsTotalCount: undefined,
   }
 
+  @bind
+  handleDownload() {
+    downloadObjectAsFile('json', this.state.globalConf, 'global_conf')
+  }
```

Comment:
1.  You only need `@bind` if you use `this`
2. If you use a button to trigger the download, you can use `<Button.Anchor href={dataUri} />` directly, without handler.